CHANGELOG.md

```diff
@@ -9,6 +9,38 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.6.0 - 2020-04-29
+
+### Added
+
+- Expose Prometheus metrics for Agent and Controller using the "/metrics" apiserver endpoint; "enablePrometheusMetrics" must be set to true in configuration.
+- Add documentation for deploying Prometheus and scraping Antrea metrics, along with sample YAML manifest.
+- Install portmap CNI by default in order to support Pods with "hostPort" set.
+- Support configurable ports for Agent and Controller apiservers.
+- Set default CPU resource requests for Antrea components in YAML manifest.
+- Add "/ovsflows" API endpoint to Agent to query OVS flows and "antctl get ovsflows" command; flows can be filtered by Pod / NetworkPolicy / OVS Table.
+- Improvements to "/networkpolicies" API endpoint and "antctl get networkpolicies" command:
+   * namespace and name parameters to filter policies
```

Comment:
Is this one new in 0.6?

Reply:
yes https://github.com/vmware-tanzu/antrea/pull/576